drivers/ethernet/eth_stm32_hal.c

```diff
@@ -347,15 +387,22 @@ static int eth_tx(const struct device *dev, struct net_pkt *pkt)
 		goto error;
 	}
 
-#if defined(CONFIG_SOC_SERIES_STM32H7X)
```

Comment:
Isn't this breaking existing NON H7 code ?

Reply:
Yes, it is. Good catch!